tests/test_vqe.py

```diff
@@ -859,8 +859,8 @@ def test_metric_tensor(self):
 
         cost = qml.VQECost(qml.templates.StronglyEntanglingLayers, hamiltonian, dev, optimize=True)
 
-        with pytest.raises(ValueError, match="Evaluation of the metric tensor is not supported"):
-            cost.metric_tensor(None)
+        args = qml.init.strong_ent_layers_uniform(2, 4)
+        cost.metric_tensor(args)
```

Comment:
Test is still WIP, but already getting an error.
`TypeError: StronglyEntanglingLayers() got multiple values for argument 'wires'`
I believe this is a gotcha from the old QNode, but I cannot see how to fix.